src/geocoders/pelias.ts

```diff
@@ -41,42 +41,25 @@ export class Pelias implements IGeocoder {
       'point.lon': location.lng
     });
     const data = await getJSON<any>(this.options.serviceUrl + '/reverse', params);
-    return this._parseResults(data, 'bounds');
```

Comment:
I don't recall the history of this but the `bbox` param isn't named `bounds`, someone has put effort into supporting alternative key names and I'm not sure why 🤷